PocoCrypto fails to build with Visual Studio projects

`1>CipherFactory.obj : error LNK2019: "public: __thiscall Poco::Crypto::EVPCipherImpl::EVPCipherImpl(class Poco::Crypto::EVPPKey const &)" (??0EVPCipherImpl@Crypto@Poco@@QAE@ABVEVPPKey@12@@Z)"public: class Poco::Crypto::Cipher * __thiscall
 Poco::Crypto::CipherFactory::createCipher(class Poco::Crypto::EVPPKey const &)" (?createCipher@CipherFactory@Crypto@Poco@@QAEPAVCipher@23@ABVEVPPKey@23@@Z) 함수에서 참조되는 확인할 수 없는 외부 기호 [C:\Users\WDAGUtilityAccount\Downloads\poco\Crypto\Cry
pto_vs170.vcxproj]
..\bin\PocoCrypto.dll : fatal error LNK1120: 1개의 확인할 수 없는 외부 참조입니다. [C:\Users\WDAGUtilityAccount\Downloads\poco\Crypto\Crypto_vs170.vcxproj]`

The offending line is: https://github.com/pocoproject/poco/blob/devel/Crypto/src/CipherFactory.cpp#L21

`2>CryptoTestSuite.obj : error LNK2019: "public: static class CppUnit::Test * __cdecl EnvelopeTest::suite(void)" (?suite@EnvelopeTest@@SAPAVTest@CppUnit@@XZ)"public: static class CppUnit::Test * __cdecl CryptoTestSuite::suite(void)" (?suit
e@CryptoTestSuite@@SAPAVTest@CppUnit@@XZ) 함수에서 참조되는 확인할 수 없는 외부 기호 [C:\Users\WDAGUtilityAccount\Downloads\poco\Crypto\testsuite\TestSuite_vs170.vcxproj]
bin\TestSuite.exe : fatal error LNK1120: 1개의 확인할 수 없는 외부 참조입니다. [C:\Users\WDAGUtilityAccount\Downloads\poco\Crypto\testsuite\TestSuite_vs170.vcxproj]`

The offending line is: https://github.com/pocoproject/poco/blob/devel/Crypto/testsuite/src/CryptoTestSuite.cpp#L24

I noticed that the Envelope.h/.cpp, EVPCipherImpl.h/.cpp, EnvelopeTest.h/.cpp files are not in the Crypto project and that the respective test classes are also not in the TestSuite project, but all the 6 files are in the filesystem.

Missing haters identified include:
Crypto -> Envelope.h, EVPCipherImpl.h
Crypto TestSuite -> EnvelopeTest.h

When I checked, it seems that there are headers missing in MSVC for projects other than Crypto.
Looks like the whole Poco project needs MSVC build check.

--- a/Crypto/Crypto_VS90.vcproj
+++ b/Crypto/Crypto_VS90.vcproj
@@ -654,6 +654,10 @@
 					>
 				</File>
 				<File
+					RelativePath=".\include\Poco\Crypto\EVPCipherImpl.h"
+					>
+				</File>
+				<File
 					RelativePath=".\include\Poco\Crypto\EVPPKey.h"
 					>
 				</File>
@@ -675,6 +679,10 @@
 				>
 				<File
 					RelativePath=".\src\CryptoException.cpp"
+					>
+				</File>
+				<File
+					RelativePath=".\src\EVPCipherImpl.cpp"
 					>
 				</File>
 				<File
@@ -733,6 +741,10 @@
 					RelativePath=".\include\Poco\Crypto\ECKeyImpl.h"
 					>
 				</File>
+				<File
+					RelativePath=".\include\Poco\Crypto\Envelope.h"
+					>
+				</File>
 			</Filter>
 			<Filter
 				Name="Source Files"
@@ -747,6 +759,10 @@
 				</File>
 				<File
 					RelativePath=".\src\ECKeyImpl.cpp"
+					>
+				</File>
+				<File
+					RelativePath=".\src\Envelope.cpp"
 					>
 				</File>
 			</Filter>

--- a/Crypto/testsuite/TestSuite_VS90.vcproj
+++ b/Crypto/testsuite/TestSuite_VS90.vcproj
@@ -583,6 +583,10 @@
 					>
 				</File>
 				<File
+					RelativePath=".\src\EnvelopeTest.h"
+					>
+				</File>
+				<File
 					RelativePath=".\src\RSATest.h"
 					>
 				</File>
@@ -608,6 +612,10 @@
 				</File>
 				<File
 					RelativePath=".\src\PKCS12ContainerTest.cpp"
+					>
+				</File>
+				<File
+					RelativePath=".\src\EnvelopeTest.cpp"
 					>
 				</File>
 				<File

